tsnet/tsnet.go

```diff
@@ -180,11 +182,19 @@ func (s *Server) start() error {
 
 	// TODO(bradfitz): start logtail? don't use filch, perhaps?
 	// only upload plumbed Logf?
+	pol := logpolicy.New(logtail.CollectionNode)
+	defer func() {
```

Comment:
I'm not sure this runs at the right time. The `start` method appears to execute and then return quickly. It doesn't seem to block while the server is still running.